cmake/modules/LXQtTranslateDesktop.cmake

```diff
@@ -72,7 +72,7 @@ function(lxqt_translate_desktop _RESULT)
         if (_translations)
             add_custom_command(OUTPUT ${_outFile}
                 COMMAND grep -v "'#TRANSLATIONS_DIR='" ${_inFile} > ${_outFile}
-                COMMAND grep -h ${_pattern} ${_translations} >> ${_outFile}
+                COMMAND grep -h ${_pattern} ${_translations} | sort >> ${_outFile}
```

Comment:
It's a matter of taste, but IMO it's better to have all keys for one language together...